src/render.js

```diff
@@ -123,16 +124,6 @@ const render = async function(
     const renderProfile = profile.start("rendering " + entryPointUrl);
 
     try {
-        // Make sure the require globals are made available to the VM context.
-        if (globals) {
-            Object.keys(globals).forEach(key => {
-                // Location is a special case.
-                if (key !== 'location') {
-                    context.window[key] = globals[key];
-                }
-            });
-        }
```

Comment:
We don't need this here anymore. It's copied to the render context creation code instead so that it occurs before scripts are loaded.